### PR TITLE
fix(compile): removing dependency on pch file from example project

### PR DESCRIPTION
**Pull request by milos1290**

### CODE REVIEW

**Comment by alexisoyama** [comment]
What is this for?
Also why are we shifting the macro check for MkNetwork? It seems to be the same but with imports.

**Comment by milos1290** [comment]
I'm removing dependency on pch file located in Example project. As for shifting macro check its either
```
#import <Foundation/Foundation.h>

#if __IPHONE_OS_VERSION_MIN_REQUIRED < 70000

#import "NSString+MKNetworkKitAdditions.h"
#import <CommonCrypto/CommonDigest.h>
```
or 
```
#import "NSString+MKNetworkKitAdditions.h"
#import <CommonCrypto/CommonDigest.h>

#if __IPHONE_OS_VERSION_MIN_REQUIRED < 70000
```

**Comment by benmarten** [comment]
I assume this fixes https://github.com/Leanplum/Leanplum-iOS-SDK/issues/28 ?

**Comment by alexisoyama** [comment]
Okay. I guess it's the same.

**Comment by milos1290** [comment]
Yeah it fixes #28. Maybe go with option 1 as it looks cleaner ?

File: Leanplum-SDK/Classes/Vendor/MKNetworkKit/Leanplum_MKNKOperationWrapper.h [hunk 1]
```diff
@@ -5,11 +5,11 @@
 //  Created by Alexis Oyama on 11/14/16.
```
Comment (alexisoyama) [inline Leanplum-SDK/Classes/Vendor/MKNetworkKit/Leanplum_MKNKOperationWrapper.h]: Did you forget to edit this file?

File: Leanplum-SDK/Classes/Vendor/MKNetworkKit/Leanplum_MKNKEngineWrapper.h [hunk 2]
```diff
@@ -5,11 +5,11 @@
 //  Created by Alexis Oyama on 11/14/16.
```
Comment (alexisoyama) [inline Leanplum-SDK/Classes/Vendor/MKNetworkKit/Leanplum_MKNKEngineWrapper.h]: Did you forget to edit this file?

File: Leanplum-SDK/Classes/Vendor/MKNetworkKit/Leanplum_MKNKOperationWrapper.h (hunk 1 above)
Comment (milos1290) [inline Leanplum-SDK/Classes/Vendor/MKNetworkKit/Leanplum_MKNKOperationWrapper.h]: yeah, my mistake.

**Comment by milos1290** [comment]
test this please

**Comment by alexisoyama** [comment]
wrapper's .h files doesn't have imports. maybe that's why the build failed?

**Comment by benmarten** [comment]
test this please

**Comment by benmarten** [comment]
@alexisoyama please have a look again ;)

**Comment by alexisoyama** [comment]
@milos1290 Can you confirm if we need to put the headers in the .h files too? 
I assume the build check is done for iOS6 too so it should be okay right? Just want to confirm first.

**Comment by milos1290** [comment]
We don't have to put it into headers file. Headers file aren't compiled

**Pull request merged**